pubsub/validators/validator.py

```diff
@@ -18,6 +18,11 @@ def validate_message(self, message):
 
 
 class SchemaValidator(BaseValidator):
+    def __init__(self, resolver=None):
+        if resolver is None:
+            resolver = RefResolver('', '')
```

Comment:
Another popular pattern for this is `self.resolver = resolver or RefResolver('', '')`